packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_CongestionPriceOracle.sol

```diff
@@ -43,30 +49,109 @@ contract OVM_CongestionPriceOracle is Ownable {
     /********************
      * Public Functions *
      ********************/
-    
+
+    /**
+     * @return Current execution price.
+     */
+    function getExecutionPrice()
+        public
+        view
+        returns (
+            uint256
+        )
+    {
+        return executionPrice;
+    }
+
     /**
-     * @return Current congestion price.
+     * Allows the owner to modify the execution price.
+     * @param _executionPrice New execution price.
      */
-    function getCongestionPrice()
+    function setExecutionPrice(
+        uint256 _executionPrice
+    )
+        public
+        onlyOwner
+    {
+        executionPrice = _executionPrice;
+    }
+
+    /**
+     * @return Current batch overhead. Represents the gas
+     * used to submit the batch on L1.
+     */
+    function getBatchOverhead()
```

Comment:
Small nit: to me `batch overhead` sounds like the overhead per batch. What we are talking about is the overhead per transaction. So to me these would be better:

- `transaction overhead`
- `transaction submission overhead`
- `batch submission overhead per transaction`
- `l1 submission overhead`
- `transaction fixed cost`

I could keep going but maybe you get what I'm trying to express here to improve the clarity of this word

Reply:
Ok yes you are right - will update